app/models/upload/UploadStatusEnum.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.upload
+
+import play.api.libs.json.{Format, JsError, JsResult, JsString, JsSuccess, JsValue}
+
+object UploadStatusEnum extends Enumeration {
+  val WAITING = Value
+  val READY = Value
+  val FAILED= Value
```

Comment:
Could you add an enum for ACCEPTED? I think Upscan could return it?
![image](https://user-images.githubusercontent.com/53828896/133608888-4c1f66b6-915e-4f63-a3d8-9b3d47e41987.png)